### ISSUE TITLE
RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.ShareIntent...

### ISSUE THREAD

**Issue by oguzkocer** [wordpress-mobile/WordPress-Android#11619]
**Sentry Url**: https://sentry.io/share/issue/bc0cd122181b44149e359dc232d89329/
**User Count**: 4
**Count**: 18
**First Release**: org.wordpress.android-14.6-rc-1
**First Seen**: 2020-04-07T15:01:12Z
**Last Release**: org.wordpress.android-14.6-rc-1
**Last Seen**: 2020-04-08T11:53:19Z
**24 Hours**: 18
**30 Days**: 18

**Stacktrace**:
```
RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.ShareIntentReceiverActivity}: android.view.InflateException: Binary XML file line #32: Binary XML file line #32: Error inflating class com.google.android.material.card.MaterialCardView
    at android.content.res.TypedArray.getColorStateList(TypedArray.java:562)
    at androidx.cardview.widget.CardView.<init>(CardView.java:127)
    at com.google.android.material.card.MaterialCardView.<init>(MaterialCardView.java:118)
    at com.google.android.material.card.MaterialCardView.<init>(MaterialCardView.java:114)
    at java.lang.reflect.Constructor.newInstance0(Constructor.java:null)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:430)
    at android.view.LayoutInflater.createView(LayoutInflater.java:656)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:798)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:738)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:869)
    at java.lang.reflect.Constructor.newInstance0(Constructor.java:null)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:430)
    at android.view.LayoutInflater.createView(LayoutInflater.java:656)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:798)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:738)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:869)
    at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:832)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:518)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:426)
    at org.wordpress.android.ui.ShareIntentReceiverFragment.onCreateView(ShareIntentReceiverFragment.java:86)


    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2804)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2875)
    at android.app.ActivityThread.-wrap12(ActivityThread.java:null)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1578)
    at android.os.Handler.dispatchMessage(Handler.java:105)
    at android.os.Looper.loop(Looper.java:156)
    at android.app.ActivityThread.main(ActivityThread.java:6617)
    at java.lang.reflect.Method.invoke(Method.java:null)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:942)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:832)
```

**Comment by khaykov** [comment]
Was able to reproduce this one. Woking on the fix.

**Comment by oguzkocer** [comment]
Fixed in #11623.
